cluster-provision/k8s/scripts/provision.sh

```diff
@@ -145,7 +150,78 @@ $reset_command
 mkdir /etc/kubernetes/audit
 
 # New configuration for kubernetes >= 1.12
```

Comment:
nit: can you update this comment or remove it?

Reply:
Done